src/en/news/blog/2025/user-accounts-dashboard/index.md

```diff
@@ -0,0 +1,155 @@
+tags:
+  - "ceph"
+  - "rgw"
+  - "user accounts"
+  - "s3"
+  - "dashboard"
+---
+
+## 1. What’s an Account & Why Use It?
+
+Instead of each S3 user operating independently, an **account groups users and roles under shared ownership**. Resources like buckets and objects are owned by the account, not individual users. This brings major advantages:
+
+- **Aggregated quotas & stats**: Track usage and enforce limits at the account level.
+- **Shared visibility**: Every user or role in the account can list and manage account‑owned buckets.
+- **Streamlined management**: Apply IAM policies centrally within the account for consistent permissions.
+
+---
+
+## 2. Roles & Responsibilities: Understanding Account Users
+
+| Role Type | What It Is | Permissions |
+|-------------------|----------------------------------------------|--------------------------------------------------------------------|
+| **Account Root User** | The master account owner, created by an admin | Full default access to account’s resources; used to define policies and create sub-users |
+| **IAM Users / Roles** | Scoped identities created by the root user | No access by default—they need explicit IAM policies to perform actions |
+
+
+## 3. User Accounts Management
+### 3.1 Account Creation
+
+The account feature is added under dashboard menu Object > Accounts. Once navigated to this page it will show the list of accounts.
+
+![Step 1: Account list page](images/account-list.png)  
+    *Figure: Step 1 - Navigating to Accounts page*
+
+Now we will click on the `Create` button for creating an account.
+
+We will be getting a form to enter values for following fields:
+
+![Step 2.1: Create Account: Enter account details](images/enter-account-form-details1.png)  
+    *Figure: Step 2.1 - Enter account details*
+
+![Step 2.2: Create Account: Enter quota details](images/enter-account-form-details2.png)  
+    *Figure: Step 2.2 - Enter quota details*
+
+After entering the details click on create Account button, post successful account creation you will be redirected to the accounts list page in which we can see the newly created account listed in table, and on expanding the table row, we can see the account details which shows the Account Quota & Bucket Quota for the account.
+
+![Step 3: Account list & details](images/account-details.png)  
+    *Figure: Step 3 - Account details*
+
+### 3.2 Link user with an Account
+
+After creating an account, the next step would be to create a account user. Lets see how its done.
+
+Navigate to the user page and click on `Create` button for creating a user.
+
+An account user can be created by two ways:  
+ - Creating a new user and linking it with an account at the same time.
+ - If you already have a user created then that user can be edited, and can be linked to an account.
+
+You will be seeing the user form in which `Link Account` will be the first input selector from which you can choose to link the user with an account, from the account list.
```

Comment:
After creating an account, the next step is to create an account user. Let’s see how it’s done.

Navigate to the Users page and click the Create button to create a new user.

An account user can be created in two ways:

    1. Create a new user and link it to an account during creation.
    2. Link an existing user to an account by editing the user.

On the user creation form, the Link Account field will be the first input selector. You can use this to choose an account from the account list and link it to the user.